src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/AvoidDeadConditionalCode_NullAnalysis.cs

```diff
@@ -6747,6 +6747,60 @@ public void M1(string path)
 
     private static void DoSomething(Stream stream) { }
 }
+",
+                LanguageVersion = CodeAnalysis.CSharp.LanguageVersion.CSharp8,
+            }.RunAsync();
+        }
+
+        [Trait(Traits.DataflowAnalysis, Traits.Dataflow.NullAnalysis)]
+        [Fact(Skip = "https://github.com/dotnet/roslyn-analyzers/issues/3685")]
+        public async Task StringIsNullOrWhiteSpace_NoDiagnostic()
```

Comment:
Can consider making this a single test with Theory/InlineData as here: https://github.com/dotnet/roslyn-analyzers/blob/8be492cb55b6e3f809f8a24d1c8b370f519534d2/src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/ValidateArgumentsOfPublicMethodsTests.cs#L5758-L5761